decentraland/unity-explorer
# fix: Add hash to GLTFComponent

## What does this PR change?

Fixes #401 
Fixes #1367 

- Adds the Hash to the `GLTFComponent`. This allows the asset to be cached by hash rather than name, therefore if two scenes have an asset with the same name they won't hit false positive when hitting the cache.
- Removes the AssetBundle by name logic. We will always have the hash available, so no need to continue looking for it at `PrepareAssetBundleLoadingParametersSystem`. 
- The logic that checked if an asset is in the scene content is now in `LoadGltfContainerSystem`. If an asset is tried to be loaded without it being in the scene content, it will fail right away.

## How to test the changes?

1. Launch the explorer
2. Go to the Game Expo and the Art Week referred in the links. Check that the assets are correct 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
